### ISSUE TITLE
5 out of 9 hunks FAILED -- saving rejects to file source4/kdc/mit-kdb/kdb_samba_policies.c.rej

### ISSUE THREAD

**Issue by zazola** [nkadel/samba4repo#10]
Trying to build the rpms for the latest version of 4.15.2, 4.14.10 and 4.13.14 (latest versions of every minor supported version of Samba) shows an error on a patch being applied:

I only changed the 'samba_version' in samba-4.14.x-srpm/samba.spec to get the specific samba version I want to build:

+ /usr/bin/cat /opt/SAMBA4REPO_4.14.10/samba-4.14.x-srpm/rpmbuild/SOURCES/samba-s4u.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
5 out of 9 hunks FAILED -- saving rejects to file source4/kdc/mit-kdb/kdb_samba_policies.c.rej
error: Bad exit status from /var/tmp/rpm-tmp.LYAVhT (%prep)

I tried to reproduce the error applying the patch out of the repo, but I'm not able to reproduce it.
I would appreciate if anyone else experienced something similar or has any insight on this particular issue.

**Comment by nkadel** [comment]
Why would you expect the patches to work between distinct versions of Samba?

The master branch, and the most recent tags, are up to v4.15.1. Why not update your checked out repos and work from there?

**Comment by zazola** [comment]
I was not expecting anything in particular (maybe I would expect something could fail in 4.15.2, as the repo is not yet prepared for that minor version), but I was just trying to compile all the latest version of those versions that were released for the security patch introduced by the Samba guys. I got the same error in all three builds separatedly, so that's why I'm reporting all at the same time.
For legacy reasons, I wanted to have the lastest version of every minor version supported, hence 4.13.14, 4.14.10 and 4.15.2 all tagged in the Samba repo.

**Comment by nkadel** [comment]
Fair, but I'd not expect tags from one release to be automatically compatible with other patches for other with new Samba releases. I also admit that I'm not supporting older releases, and trying to stay caught up, with 4.15.1 as the latest. 4.15.2, unfortunately, has some python 3.8 dependencies I've not successfully resolved.

**Comment by zazola** [comment]
OK, I understand. But 4.14.10 I should be able to build, right?

**Comment by nkadel** [comment]
Yeah. CentOS 8 stream has a samba415 repo, with the dependencies, but without the Heimdal based and more reliable kerberos.
